sdk/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub mod keystore;
 pub mod random;
 
 /// Blockchain Events Watcher Module
+#[cfg(any(feature = "std", feature = "wasm"))]
```

Comment:
Same here as the comment below. If it can work in `std` and `no_std` we shouldn't need to specify feature flags here. It should be configured in the TOML by adding the relevant packages to the `std` feature.

For example
````
std = ["package_that_works_in_no_std_and_std/std", ...]
```

Reply:
@drewstone That module has TODOs for more features anyway: https://github.com/webb-tools/gadget/blob/e910dc30a554a4a946328e9252166e8db5665a16/sdk/src/events_watcher/error.rs#L6

It'd be easier to handle that separation in its own PR, which I can do later.